### ISSUE TITLE
Nested values?

### ISSUE THREAD

**Issue by jjatria** [key-amb/perl5-AWS-CLI-Config#3]
In the documentation for the AWS config file there is a section on [nested values](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#nested-values) with the following example:

```
[profile testing]
aws_access_key_id = foo
aws_secret_access_key = bar
region = us-west-2
s3 =
  max_concurrent_requests=10
  max_queue_size=1000
```

I do have some projects in which nested values would be useful, but I don't use them because I can't get them from Perl, since as far as I can tell, there are no distributions on CPAN that can correctly parse this (not even Paws).

Being able to parse them would be great, but it might be too much of a hassle...

**Comment by key-amb** [comment]
Looks a hard work.

Actually, currently I don't use this library, so I can add you as a comaint on CPAN if you like.

**Comment by jjatria** [comment]
If you are not interested in maintaining this any further, then sure, I can help. My PAUSE ID is JJATRIA.

I might make an attempt at parsing this at some point.

**Comment by key-amb** [comment]
Fixed by #4

**Comment by key-amb** [comment]
@jjatria 

Thank you for your reply.
I've added you as a comaint of this module at CPAN.
And I've also added you as a collaborator of this repo on GitHub.com .

You can ship to CPAN from your forked repository, off course. It's your choice.
My only wish is that you won't drop my name from AUTHORs of this module.

Thanks,
